tests/pairedomics/test_podp_antismash_downloader.py

```diff
@@ -158,33 +164,32 @@ def test_missing_id(download_root, extract_root, genome_status_file):
     genome_records = [{
         "genome_ID": {
             "genome_type": "genome",
-            "JGI_Genome_ID": "",
-            "RefSeq_accession": ""
+            "JGI_Genome_ID": "2515154188",
```

Comment:
I think that the point of this record (and test) was to test a case in which there are no accession IDs, that's why their fields were empty.

Reply:
Aha! I will change it back.